attractions/time-picker/time-picker.svelte

```diff
@@ -121,29 +129,46 @@
       <div class="shown-on-focus">
         <Button noRipple on:click={() => focus = false}>close the time picker</Button>
       </div>
-      <Label>Hours</Label>
+      <Label>
+        <slot name="hours-label">
+          <Label>Hours</Label>
+        </slot>
+      </Label>
       <div class="column">
-        {#each hourValues as value}
-        <Button on:click={() => setHours(value + 12 * (f12hours && currentAmPm === 'PM' ^ value === 12))}>
-          {value.toString().padStart(2, '0')}
-        </Button>
+        {#each hourValues as hour}
+          <Button
+            on:click={() => setHours(hour + 12 * (f12hours && (currentAmPm === 'PM') ^ (value === 12)))}
+            selected={matchesCurrentHour(hour, value)}>
```

Comment:
Also I haven't exactly mentioned it earlier (and Prettier is still not set up to guide you in this), but we prefer the closing angle bracket on a separate line:
```svelte
<Button
  props
>
```

Reply:
ok, will fix. yeah I have been using prettier for more than two years in all my work projects. Sort of forgot what it is like to develop js/ts without it on the frontend (backend I still format by hand unfortunately).